### DISCUSSION TITLE
Getting "info" from logs

### DISCUSSION THREAD

**Discussion by michaelwooley** [PrefectHQ/prefect#4153] [General]
I noticed that I could potentially get more info from my logs by passing the `-i` flag:

```bash
prefect get logs -n rebel-mouse -i
```

However, this returns many _empty_ lines. 

Excluding the `-i` flag we get useful information:

```bash
prefect get logs -n rebel-mouse
```

In  a sense, this makes sense. **When I inspect the `logs` table in the database, I noticed that the `info` column is always null.** From [here](https://github.com/PrefectHQ/prefect/blob/master/src/prefect/cli/get.py#L410), we can see that the first command is just printing these empty lines.

- **Question.** Is the lack of `info` due to a configuration option? If so, what one?
- **Suggestion.** When passing the `-i` flag, print the other columns along with the `info` column.

I would be happy to submit a PR related to this, an issue in prefect-server (nothing popped out to me just now), or a docs update. Just let me know.

---

_Other details._ 

- I am running prefect locally. 
- My python version is 3.9 ([which I know is not necessarily fully-supported](https://github.com/PrefectHQ/prefect/issues/3453)).
- This occurs with both the docker and local agents.

Output of `prefect diagnostics`

```json
{
  "config_overrides": {},
  "env_vars": [],
  "system_information": {
    "platform": "Linux-5.9.16-1-MANJARO-x86_64-with-glibc2.33",
    "prefect_backend": "server",
    "prefect_version": "0.14.9",
    "python_version": "3.9.1"
  }
}
```

**Comment by zanieb** [comment]
Hi @michaelwooley, thanks for the report!

We actually dropped storing the `info` field a while back because it's unspecced and unused data that was increasing log storage and network usage. I think the proper solution here is to deprecate the `-i` flag since (as you've noted) it doesn't do anything.

**Reply by michaelwooley** [reply]
@madkinsz  okay, would you accept a PR to this effect? If so, would you like an issue created first?

**Reply by zanieb** [reply]
We'd definitely appreciate a PR :) no need to create an issue first. I'd take a look at some of our other deprecation message so that it's consistent with what we'd typically do.
